e2e-tests/api-tests/build.gradle.kts

```diff
@@ -16,6 +16,8 @@ dependencies {
     testImplementation(testFixtures(project(":spi:identity-hub-spi")))
     testImplementation(libs.nimbus.jwt)
     testImplementation(libs.jakarta.rsApi)
+
+    testRuntimeOnly(project(":launcher"))
```

Comment:
is this on purpose? and if so, is this just to add a task dependency from `:e2e-tests:api-tests` -> `:launcher`?

Reply:
On my windows laptop, it does not work without that addition.